storybook/src/stories/person/select-example.stories.tsx

```diff
@@ -1,23 +1,17 @@
+import { useState, useEffect } from 'react';
 import { Meta, Story } from '@storybook/react';
-import { PersonProvider, PersonAccountType, PersonSelect, PersonSelectProps } from '@equinor/fusion-react-person/src';
+import { PersonInfo, PersonProvider, PersonSelect, PersonSelectProps } from '@equinor/fusion-react-person/src';
 import { createResolve } from './resolve-mock/person-resolve-mock';
+import { PersonSelectEvent } from '../../../../packages/person/src/PersonSelect';
 
 export default {
   title: 'Examples/Person/Person Select',
   component: PersonSelect,
   argTypes: {
-    id: {
-      type: 'string',
-      description: 'HTML ID for component',
-    },
     placeholder: {
       type: 'string',
       description: 'TextInput placeholder string',
     },
-    initialText: {
-      type: 'string',
-      description: 'First item in dropdown before any search is done',
-    },
```

Comment:
same as above? initial text should be included?

Reply:
initialText is a unused property.